src/OpenAL/OpenToolkit.OpenAL/ALDevice.cs

```diff
@@ -0,0 +1,54 @@
+﻿//
+// ALDevice.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+    /// <summary>
+    /// Opaque handle to an OpenAL device.
+    /// </summary>
+    public struct ALDevice : IEquatable<ALDevice>
+    {
+        public static readonly ALDevice Null = new ALDevice(IntPtr.Zero);
+
+        public IntPtr Handle;
```

Comment:
Same as above in ALContext maybe. Maybe.

Reply:
There is an implicit cast now :)